api/services/userService.js

```diff
@@ -0,0 +1,80 @@
+const {userDao} = require("../models");
+const jwt = require('jsonwebtoken');
+const axios = require('axios');
+
+const login = async (code) => {
+    const getToken = await axios.post("https://kauth.kakao.com/oauth/token",
+        {
+            grant_type: "authorization_code",
+            client_id: process.env.CLIENT_ID,
+            redirect_uri: "http://localhost:3000/auth/kakao/callback",
+            code: code
+        },
+        {
+            headers: {
+                'Content-Type': 'application/x-www-form-urlencoded;charset=utf-8'
+            }
+        });
+
+    const accessToken = getToken.data.access_token
+    console.log(`accessToken: ${accessToken}`);
+
+    const getKakaoUserData = await axios.get(
+        `https://kapi.kakao.com/v2/user/me`,
+        {
+            headers: {
+                Authorization: `Bearer ${accessToken}`,
+                'Content-Type': 'application/x-www-form-urlencoded',
+            },
+        }
+    );
+
+    const kakaoId = getKakaoUserData.data.id;
+    const {nickname} = getKakaoUserData.data.properties;
+    const {email} = getKakaoUserData.data.kakao_account;
+
+    const isExist = await userDao.checkRegistered(kakaoId);
+
+    const response = {
+        accessToken: '',
+        statusCode: 0,
+    }
```

Comment:
```suggestion
    let response = {
        accessToken: '',
        statusCode: 0,
    }
```
아래의 코드 로직에 따라 object의 value가 변할 수 있는 것이라면 let으로 선언하는것이 올바르게 보입니다.